.eslintrc.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
   rules: {
     'object-shorthand': 'off',
     'prefer-destructuring': 'off',
+    'no-unused-expressions': 'off',
```

Comment:
[nit] Just curious why you've disabled this one?  Was there a specific need?  Generally, I find this to be nice in that it keeps code clean over time.

Reply:
yes, thanks for asking. With the nightwatches new expect api (based off of chai) this error comes up a lot. I went ahead and added the chai eslint plugin, since i think that makes more sense instead of adding rules one by one.